hypershift-operator/controllers/nodepool/capi.go

```diff
@@ -1375,3 +1382,7 @@ func (r *NodePoolReconciler) getMachinesForNodePool(ctx context.Context, nodePoo
 
 	return sortedByCreationTimestamp(machinesForNodePool), nil
 }
+
+func (c *CAPI) reconcileORCResources(ctx context.Context) error {
+	return openstack.ReconcileOpenStackImageCR(ctx, c.Client, c.CreateOrUpdate, c.hostedCluster, c.controlplaneNamespace, c.releaseImage)
```

Comment:
It's because ORC isn't logging anything steady state at V(2). I think we should change this in ORC, but for now it might be better just to bump CAPO's log level from 2 to 3 or 4.